### DISCUSSION TITLE
Naming of the website created using GitHub Pages

### DISCUSSION THREAD

**Discussion by AnimishY** [community/community#22706] [Other Feature Feedback, Questions, & Ideas]
<p>Hey All at GitHub Community</p>
<p>I have seen many people using GitHub Pages and their overall URL is in this form -</p>
<p><a href="http://username.github.io" class="onebox" target="_blank" rel="nofollow noopener">username.github.io</a></p>
<p>But when I tried for the same, My URL is coming out to be as -</p>
<p><a href="http://username.github.io/reponame" class="onebox" target="_blank" rel="nofollow noopener">username.github.io/reponame</a></p>
<p>So, I wanted to know how I can also get a URL like that which makes it is easier for people to visit my website instead of writing long URLs?</p>
<p>Also, if it does matters for this problem, the repository is under an organization account.</p>
<p>Thank You and Waiting for a quick and ideal solution!<br>
Animish Yadav (<a class="mention" href="/animishy">@AnimishY</a>)</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/animishy">@AnimishY</a>,</p>
<p>To create a page for an organization, you need to create a repo named <code>&lt;organization&gt;.github.io</code> under that account.  <code>&lt;organization&gt;</code> should be the name of your organization. This also works the same for user sites, however, <code>&lt;organization&gt;</code> should be your GitHub username and the repo needs to be created under your user account.</p>
<p>See <a href="https://docs.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites" rel="nofollow noopener">Types of GitHub Pages sites</a> for more information.</p>

**Comment by AnimishY** [comment]
<p>Hello <a class="mention" href="/SethClydesdale">@SethClydesdale</a></p>
<p>Still a question, when I name the repo as . <a href="http://github.io" rel="nofollow noopener">github.io</a>, do I need to include rhe brackets?</p>
<p>So Overall, repo name should be same as the URL I need without any /repo</p>
<p>Thank you</p>

**Comment by tajmone** [comment]
<p>The <code>&lt;</code> and <code>&gt;</code> brackets are just conventional placeholders to inform you that you need to replace the sample text with your own. So, in case of <code>&lt;organization&gt;.github.io</code> you’ll have to replace <code>&lt;organization&gt;</code> with your organization name (as is, <em>without</em> brackets).</p>
<p>Also, the angle brackets characters <code>&lt;</code> and <code>&gt;</code> (aka <em>less-then</em> and <em>greater-then</em> symbols) are not valid URL characters, so you won’t ever find them in actual URLs:</p>
<aside class="onebox stackexchange">
  <header class="source">
      <a href="https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid" target="_blank" rel="nofollow noopener">stackoverflow.com</a>
  </header>
  <article class="onebox-body">
      <a href="https://stackoverflow.com/users/186008/good" target="_blank" rel="nofollow noopener">
    <img alt="good" src="https://user-images.githubusercontent.com/11064292/181072656-d288160d-4f20-4d19-af11-e7f1b5949629.png" class="thumbnail onebox-avatar" width="128" height="128">
  </a>
<h4>
  <a href="https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid" target="_blank" rel="nofollow noopener">Which characters make a URL invalid?</a>
</h4>

<div class="tags">
  <strong>validation, url, rfc3986</strong>
</div>

<div class="date">
  asked by
  
  <a href="https://stackoverflow.com/users/186008/good" target="_blank" rel="nofollow noopener">
    good
  </a>
  on <a href="https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid" target="_blank" rel="nofollow noopener">01:10PM - 10 Oct 09 UTC</a>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
